amf-client/shared/src/test/resources/org/raml/api/v10/full/model.json

```diff
@@ -294,7 +295,8 @@ annotationTypes:
         type: string
         required: false
     headers:
-      header-one: {}
+      header-one:
```

Comment:
Default type "string"

Reply:
At input.raml (source) the header is:
`        headers: &headersGet
            header-one:
            header-two:
                displayName: The Second`
Type is synthesized, so have sense to not emit it